### PR TITLE
Chemistry data conversion tool

### PR DESCRIPTION
**Pull request by cgranade**
This PR merges in feature work on a new command-line tool to convert between different electronic structure problem formats, `qdk-chem convert`.

### CODE REVIEW

File: Chemistry/src/DataModel/Serialization/Broombridge/BroombridgeData.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
+#nullable enable
```
Comment (msoeken) [inline Chemistry/src/DataModel/Serialization/Broombridge/BroombridgeData.cs:3]:
```suggestion

#nullable enable

```

File: Chemistry/src/DataModel/Serialization/Broombridge/BroombridgeSerializer.cs [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+                    }
+                );
+        }
+
+        public static void Serialize(TextWriter writer, IEnumerable<ElectronicStructureProblem> problems)
+        {
+            Serializers.SerializeBroombridgev0_2(
+                new Broombridge.V0_2.Data
+                {
+                    // TODO: fix additional properties by converting IEnumerable<ESP> to
```
Comment (msoeken) [inline Chemistry/src/DataModel/Serialization/Broombridge/BroombridgeSerializer.cs:57]: Just pointing out to the `TODO`

File: Chemistry/src/Tools/Extensions.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (msoeken) [inline Chemistry/src/Tools/Extensions.cs:1]:
```suggestion
// Copyright (c) Microsoft Corporation.
```

File: Chemistry/src/Tools/Normalize.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (msoeken) [inline Chemistry/src/Tools/Normalize.cs:1]:
```suggestion
// Copyright (c) Microsoft Corporation.
```

File: Chemistry/src/Tools/Program.cs [hunk 5]
```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (msoeken) [inline Chemistry/src/Tools/Program.cs:1]:
```suggestion
// Copyright (c) Microsoft Corporation.
```

File: Chemistry/tests/TestData/FciDump/README.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# FCIDUMP Examples #
+
+Example FCIDUMP files to test the FCIDUMP-BroomBridge file converter.
+
+## File Descriptions ##
+- **[ethylene_ccpvdz.fcidump](ethylene_ccpvdz.fcidump)**:
+    Ethylene molecule in cc-pVDZ basis, 10 orbitals and 16 electrons.
```
Comment (msoeken) [inline Chemistry/tests/TestData/FciDump/README.md:7]:
```suggestion
    Ethylene molecule in cc-pVDZ basis, 10 orbitals, and 16 electrons.
```

File: Chemistry/tests/TestData/FciDump/README.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+- **[h2o_ccpvdz.fcidump](h2o_ccpvdz.fcidump)**:
+    Water molecule in cc-pVDZ basis, 9 orbitals and 10 electrons.
```
Comment (msoeken) [inline Chemistry/tests/TestData/FciDump/README.md:9]:
```suggestion
    Water molecule in cc-pVDZ basis, 9 orbitals, and 10 electrons.
```

File: Chemistry/tests/TestData/FciDump/README.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+- **[h2_anorccmb.fcidump](h2_anorccmb.fcidump)**:
+    Hydrogen molecule in ANO-RCC minimal basis, 2 orbitals and 2 electrons.
```
Comment (msoeken) [inline Chemistry/tests/TestData/FciDump/README.md:11]:
```suggestion
    Hydrogen molecule in ANO-RCC minimal basis, 2 orbitals, and 2 electrons. 
```

**Comment by cgranade** [comment]
Thanks for the review, @msoeken and @dgwvfxn! Since this is a protected feature branch, I'll address your suggestions in another PR shortly and merge that with this one. Thanks!

**Pull request merged**